src/awsexplorer/activation.ts

```diff
@@ -67,20 +66,23 @@ async function registerAwsExplorerCommands(
         callback: async () => {
             await ext.awsContextCommands.onCommandShowRegion()
             await recordNumberOfActiveRegionsMetric(awsExplorer)
-        }
+        },
+        telemetryName: 'Command_aws.showRegion'
```

Comment:
"Command_<commandname>" seems to be the majority. Thoughts on making that the default telemetryName in registerCommand? It could reduce the risk of name divergence

Reply:
I'd like to try to get away from namespace as much as possible. New metrics should not use `Command`, so as long as this PR is correct, we will not have an issue.